lib/workos/castable.ex

```diff
@@ -0,0 +1,34 @@
+defmodule WorkOS.Castable do
```

Comment:
This module defines a behavior related to data transformation. It's going to be used by the HTTP client implementation in order to cast responses to others structs.